src/routes/webhooks/apple.ts

```diff
@@ -57,6 +72,66 @@ interface AppleNotificationRequest {
   signedPayload: string;
 }
 
+const productIdToCycle = {
+  annualSpecial: SubscriptionCycles.Yearly,
+  annual: SubscriptionCycles.Yearly,
+  monthly: SubscriptionCycles.Monthly,
+};
+
+const renewalInfoToSubscriptionFlags = (
+  data: JWSRenewalInfoDecodedPayload,
+): UserSubscriptionFlags => {
+  const cycle =
+    productIdToCycle[data.autoRenewProductId as keyof typeof productIdToCycle];
+
+  if (isNullOrUndefined(cycle)) {
+    logger.error({ data }, '[Apple StoreKit]: Invalid auto renew product ID');
```

Comment:
btw. for paddle we send `{ data, type: 'paddle' }` easier to filter paddle specific logs instead from log strings, so maybe instead of `[Apple StoreKit]` we can also send `{ data, type: 'storekit }`?

Reply:
👍

Reply:
Replaced